src/components/VersesDropdown/index.js

```diff
@@ -2,9 +2,21 @@ import React, { Component, PropTypes } from 'react';
 import DropdownButton from 'react-bootstrap/lib/DropdownButton';
 import MenuItem from 'react-bootstrap/lib/MenuItem';
 import { Link } from 'react-scroll';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { setCurrentAyah } from '../../redux/modules/ayahs';
 
 const style = require('./style.scss');
 
+@connect(
```

Comment:
we should really avoid having `@connect` in this dropdown. It should come from `Surah/index.js` as a prop